guides/framework-guides/framework-remix.mdx

```diff
@@ -212,25 +213,25 @@ import {
 
 Next, create a Novu workflow with an email step. This code-first notification workflow approach makes it easy for product teams to modify notification content.
 
-Within the `app` directory, create an `echo` folder and add a `client.ts` file to it. Copy/paste the code below to the recently created file.
+Within the `app` directory, create an `novu` folder and add a `workflows.ts` file to it. Copy/paste the code below to the recently created file.
 
 ```jsx
-// app/echo/client.ts
+// app/novu/workflows.ts
 
-import { Echo } from '@novu/echo';
+import { Client } from '@novu/framework';
 import { renderEmail } from '~/emails/vercel-invite-user';
 
-export const echo = new Echo({
+export const client = new Client({
```

Comment:
Ditto

Reply:
We could remove client setup here, since this configuring is the same as the defaults if the client isn't supplied